app/build.gradle

```diff
@@ -1,127 +1,127 @@
 apply plugin: 'com.android.application'
 apply plugin: 'org.jetbrains.kotlin.android'
 
+java {
+  toolchain {
```

Comment:
Setting language version in the toolchain is not very common. Or at least haven't seen it before. Would it be reasonable to add some comment about it explaining why it is done like this.

Reply:
Added reference to the documentation — [3b4f7f9](https://github.com/salemove/android-sdk-widgets/pull/1044/commits/3b4f7f9c236ce3be0660d7eb548b3ea78180b438)